src/content/troubleshooting/_index.en.md

```diff
@@ -58,7 +58,7 @@ If you are able to connect to remote service by using ServiceIP or globalIp, but
 This is good time to familiarize yourself with [Service Discovery Architecture](../architecture/service-discovery/) if you haven't already.
 
 ##### Check ServiceExport for your Service
-For a Service to be accessible across clusters, you must first create a `ServiceExport` resource. Make sure the `ServiceExport` resource exists and has the same name and namespace as the Service you're trying to export.
+For a Service to be accessible across clusters, you must first create a `ServiceExport` resource. Make sure the `ServiceExport` resource exists and has the same name and namespace as the Service you're trying to export. If it exists, check if the status of ServiceExport is `Exported'. If not, check the status and the message to know the exact reason.
 
 ```kubectl get serviceexport -n <service-namespace> <service-name>```
```

Comment:
Looking at https://github.com/submariner-io/lighthouse/pull/187
Can we also check the status of ServiceExport with with kubectl rollout or wait commands ?

Reply:
The below commands show you the status, but don't think rollout command supports it.
```
kubectl --context cluster2  describe serviceexport nginx
kubectl --context cluster2  get serviceexport nginx -o yaml
```